cpp/CMakeLists.txt

```diff
@@ -392,7 +392,6 @@ add_library(
   src/io/csv/reader_impl.cu
   src/io/csv/writer_impl.cu
   src/io/functions.cpp
-  src/io/json/byte_range_info.cu
```

Comment:
Accidental deletion?

Reply:
This file had the implementation for `find_first_delimiter` which is invoked only in `read_json.cu`. The function is moved now and `byte_range_info.cu` can be deleted.

Reply:
Then please delete `byte_range_info.cu`. Do not leave it dangled.

Reply:
Oops, missed that! I have removed the file now.